src/common/common.gypi

```diff
@@ -26,7 +26,12 @@
         },
       }],
       ['tizen == 1', {
-        'defines': ['TIZEN']
+        'defines': ['TIZEN'],
+        'variables': {
+          'packages': [
+            'dlog',
+          ],
+        },
```

Comment:
I would let this new dependency protected by condition of `extension_build_type== "Debug"`